### ISSUE TITLE
Missing panel constructor definitions in common.d.ts

### ISSUE THREAD

**Issue by ryantxu** [grafana/grafana-sdk-mocks#1]
I am new to this, so I could be all wrong 😄 

I am trying to convert [influx-admin-panel](https://github.com/NatelEnergy/grafana-influx-admin) to typescript.  In the call to super() in [my constructor](https://github.com/NatelEnergy/grafana-influx-admin/blob/master/src/module.ts#L46)

I got the error:
```
error TS2346: Supplied parameters do not match any signature of call target.
```

Everything works when I add the following line to common.d.ts:
 
```
declare module 'app/plugins/sdk' {
  export class PanelCtrl{
   ...
    constructor($scope: any, $injector: any);   <<< Add this line
```

Am I doing something wrong?  or is the config missing constructor params?

**Comment by jonyrock** [comment]
Hey @ryantxu You may try my typescript template meanwhile 
https://github.com/CorpGlory/grafana-plugin-template-webpack-typescript
Sorry @daniellee :)

**Comment by daniellee** [comment]
Totally possible that I am missing something here. I've only used it with a data source plugin so far. My plan is to convert the Worldmap and Clock panels to TypeScript but haven't made time for that as of yet. 

I will have a look at @jonyrock repo and see if I can use that instead for the Typings.

**Comment by jonyrock** [comment]
@daniellee it is pretty much the same except I install typings to `node_modules/@types/grafana` folder. Then I able to write something like `import { PanelCtrl } from 'grafana/app/plugins/sdk'`
And then I resolve `grafana/app/path` to `app/path`

But I think it's a bit off topic. Sorry.

**Comment by daniellee** [comment]
@ryantxu Is this fixed by #3?

**Comment by ryantxu** [comment]
works now.  thanks!
